dandischema/model_types.py

```diff
@@ -422,169 +422,169 @@
             "rdfs:subClassOf": {"@id": "schema:Enumeration"},
         },
         {
-            "@id": "dandi:Author",
+            "@id": "dandirole:Author",
```

Comment:
so it is `dandi:RestrictedAccess` (thus `Access` is folded into value) but `dandirole:Author` and not e.g. `dandi:AuthorRole` ... ideally should be consistent

Reply:
let me think about this. it is indeed our namespace so we can control.

Reply:
i moved everything for datacite to a `dcite` prefix. unfortunately they don't have a url so i just used a dandi namespace for it.